.github/workflows/validate.yml

```diff
@@ -14,6 +14,9 @@ jobs:
     runs-on: ubuntu-20.04
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/setup-go@v2
+        with:
+          go-version: 1.x # Latest stable
```

Comment:
Perhaps cleaner to combine this commit with the actual PR that updates to v3, that way the change can be reviewed together with that 

Also wondering if `1.x` should better be tied to actual versions we're testing against; I've had cases where Go updates caused the linter to fail (or golangci-lint not yet being compatible with the latest version on "day 1"), and it's nice to have changes related to that together with the actual go update / addition in CI)

Reply:
Note that it's already possible (and a good idea) to specify the Go version to use for golangci-lint, as it's currently dependent on the version you install; see https://github.com/containerd/containerd/commit/534685f9558cc67125156ba2fbdd0e0ade74cc33, which did so for containerd

Reply:
So, basically, `golangci/golangci-lint-action@v3` makes `skip-go-installation: true` the only way.

I am using `go-version 1.x` to avoid updating go every time the new release is becoming "latest". Quoting the commit message:

> In our case, we need at least go 1.16, and can use 1.17 or 1.18 (when
> it is released), but we don't set it explicitly here to ease the
> maintenance burden (1.x should point to the latest stable).

You're right that we're risking something to be broken here when the go version is changed.

Updated to follow your recommendations; PTAL @thaJeztah